src/devices/esp8266.c

```diff
@@ -85,9 +85,12 @@ static bool sparse_urc_cb(char* msg)
 {
         msg = strip_inline(msg);
 
-        if (STR_EQ(msg, "WIFI DISCONNECT")) {
-                if (state.hooks.client_wifi_disconnect_cb)
-                        state.hooks.client_wifi_disconnect_cb();
+        /* Handle and ignore NO-OP URCs */
```

Comment:
I should remove that comment.  Originally I was going to ignore them but then I settled on the "state changed" event callbacks to make things consistent and went with that model.